docs/source/sdks.rst

```diff
@@ -6,10 +6,9 @@ SDK API Reference
 Python
 ------
 
-.. toctree::
+- `Transaction Processor <sdks/python_sdk/processor.html>`_
 
-   sdks/python_sdk/processor.rst
```

Comment:
The python docs should not have changed, since we didn't move those.  The existing links work in current master

Reply:
Agreed.  The Python docs are auto-generated, and they're still in the same place.  I had to change the extension from "rst" to "html" when I removed the toc construct, but that was just to get the Sphinx docs to generate the correct (same) URLs for the python docs.  

The links still point to the published SDK files on master:  
https://sawtooth.hyperledger.org/docs/core/nightly/master/sdks/python_sdk/processor.html
https://sawtooth.hyperledger.org/docs/core/nightly/master/sdks/python_sdk/sawtooth_signing.html

Reply:
When are they generated?

Reply:
They seem to be generated during a doc build, but I don't know anything about the exact mechanics and timing.  Perhaps @vaporos can answer this question?

Reply:
I have learned that my change to `docs/source/conf.py` is a bad idea, because it turns off the auto-generation of the python SDK docs.  However, if I remove the detailed contents list for the Python SDK, the doc build fails because`sdks/python_sdk/processor.rst` and `sdks/python_sdk/sawtooth_signing.rst` aren't referenced in a TOC.  

Sphinx has an `:orphan:` metadata directive that would ignore this error, but I can't add it to the files because they're auto-generated.  I will need help to figure out where (and how) these auto-generated files are created:  

https://sawtooth.hyperledger.org/docs/core/nightly/master/sdks/python_sdk/processor.html

https://sawtooth.hyperledger.org/docs/core/nightly/master/sdks/python_sdk/sawtooth_signing.html

Reply:
More importantly, I need to find out if it's possible to insert or prepend an `:orphan:` line in these auto-generated files.